ui/job-view/pushes/JobButton.jsx

```diff
@@ -88,11 +88,12 @@ export default class JobButtonComponent extends React.Component {
       visible,
       id,
       job_type_symbol,
-      btnClass,
+      resultStatus,
     } = job;
 
     if (!visible) return null;
     const runnable = state === 'runnable';
+    const btnClass = getBtnClass(resultStatus, failure_classification_id);
```

Comment:
Setting the ``btnClass`` on ``addAggregateFields`` didn't always get updated at the right time after classifying a job.  This returns it to just getting it on the fly during render.